repo/packages/C/cassandra/0/config.json

```diff
@@ -91,6 +97,11 @@
           "type": "string",
           "default": "dcos"
         },
+        "binary": {
+          "description":"Cassandra binary",
+          "type":"string",
+          "default":"http://downloads.mesosphere.com/cassandra-mesos/artifacts/0.2.0-1/cassandra-mesos-0.2.0-1.tar.gz"
+        },
```

Comment:
I don't recommend this. I this going to break our packaging for local universe (datacenters without access to that URL) and service upgrade.